Snowflake.Data/Core/Session/SessionPool.cs

```diff
@@ -567,5 +594,10 @@ internal List<long> GetIdleSessionsStartTimes()
                 return _idleSessions.Select(s => s.GetStartTime()).ToList();
             }
         }
+
+        internal String PoolIdentification() =>
+            IsMultiplePoolsVersion()
+                ? " [pool: " + _connectionStringWithoutSecrets + "]"
+                : "";
```

Comment:
why not to have same info for old pool?

Reply:
because in case of the old pool there is only one pool so it doesn't need to be identified

Reply:
you're right cause this is just a cache with multiple different connections strings, closing this comment